runtime/databricks/automl_runtime/forecast/pmdarima/training.py

```diff
@@ -35,7 +35,8 @@ class ArimaEstimator:
     """
 
     def __init__(self, horizon: int, frequency_unit: str, metric: str, seasonal_periods: List[int],
-                 num_folds: int = 20, max_steps: int = 150, exogenous_cols: Optional[List[str]] = None) -> None:
+                 num_folds: int = 20, max_steps: int = 150, exogenous_cols: Optional[List[str]] = None,
```

Comment:
Since you are at it, might as well change the style to 

```
exogenous_cols: List[str] | None
```

See https://peps.python.org/pep-0604/

Reply:
Fixed.

Reply:
It seems the python version of this repo does not support the new style.
<img width="696" alt="image" src="https://github.com/user-attachments/assets/1120f3e7-eebf-4b4f-a5ce-4d9aede976f5">
revert it back to Optional